examples/oft-alt/layerzero.config.ts

```diff
@@ -0,0 +1,60 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+
+import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const sepoliaContract: OmniPointHardhat = {
+    eid: EndpointId.SEPOLIA_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const fujiContract: OmniPointHardhat = {
+    eid: EndpointId.AVALANCHE_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const amoyContract: OmniPointHardhat = {
+    eid: EndpointId.AMOY_V2_TESTNET,
+    contractName: 'MyOFT',
```

Comment:
Can you update contractName here to be MyOFTAlt or MyOFTAdapterAlt based on the config you're wanting as default?

Reply:
Ah yes I'll do that first thing tmrw. Thanks for catching that